examples/experimental/test/end_to_end/candid_rpc/robust_imports/src/azle_coverage/index.ts

```diff
@@ -167,11 +168,11 @@ export const buyHoneydew = honeydew(() => {
 });
 
 export const keepIlamaClean = ilama(() => {
-    console.info(`Method "${lemon.methodName()}" was called`);
-    if (lemon.methodName() === 'dirtyIlama') {
+    console.info(`Method "${methodName()}" was called`);
+    if (methodName() === 'dirtyIlama') {
```

Comment:
Don't you think that in the spirit of robust imports we should rename methodName to melon, acceptMessage to artichoke, etc?